src/main/main.ts

```diff
@@ -39,7 +39,12 @@ let argv: string[] = [];
  */
 export async function onReady() {
   await onFirstRunMaybe();
-  if (!isDevMode()) process.env.NODE_ENV = 'production';
+  if (!isDevMode()) {
+    process.env.NODE_ENV = 'production';
+  } else {
+    const { devtron } = await import('@hitarth-gg/devtron');
+    await devtron.install();
```

Comment:
```suggestion
    await devtron.install().catch(err => {
      console.warn(`cannot install devtron: ${err.message}`)
    });
```

When the installation fails, the entire application should not be rendered unable to open.

Reply:
I believe `devtron.install()` is made to [catch any errors itself and log](https://github.com/electron-userland/devtron/blob/7e10e55d26b7e112fc27acc04a928f90456bea49/src/index.ts#L132-L134) to prevent this, but sure, we can be defensive here just in case. 👍